src/components/filters/MeasureValueFilter/DropdownBody.tsx

```diff
@@ -124,18 +134,49 @@ class DropdownBodyWrapped extends React.PureComponent<IDropdownBodyProps, IDropd
         return null;
     };
 
+    private renderTreatNullAsZeroOption = () => {
```

Comment:
I would prefer to move it to a separate file. Possibly the whole input section can be extracted.